include/psen_scan_v2/monitoring_frame_deserialization.h

```diff
@@ -71,8 +88,25 @@ class FixedFields
   FromTheta from_theta_;
   Resolution resolution_;
 };
+/**
+ * @brief Namespace containing all information/types needed to describe the content of the additional information field
```

Comment:
```suggestion
 * @brief Contains all information and types for additional information field
```

Reply:
All namespace descriptions start with `Namespace containing` - so you could either keep it here as well or search-replace all of them ;-) ?